web/src/containers/StateDashboard.js

```diff
@@ -28,7 +29,15 @@ const StateDashboard = (
   },
   { global = window } = {}
 ) => {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
This is unrelated to this PR, but your use of hooks here reminds me: Should we write up something in the wiki about why we don't use hooks more extensively?

Reply:
Oh, good idea! We use some of them pretty extensively at this point, but avoid others. Definitely worth explaining why!